subsys/net/lib/nrf_cloud/coap/src/nrf_cloud_coap_transport.c

```diff
@@ -147,6 +151,32 @@ static int server_resolve(struct sockaddr_in *server4)
 	return 0;
 }
 
+static int add_creds(void)
+{
+	int err = 0;
+
+#if defined(CONFIG_NRF_CLOUD_COAP_WIFI_ADD_CREDS_FROM_FILE)
+	err = tls_credential_add(CONFIG_NRF_CLOUD_SEC_TAG,
+				TLS_CREDENTIAL_CA_CERTIFICATE,
+				ca_certificate,
+				sizeof(ca_certificate));
+	if (err < 0) {
+		LOG_ERR("Failed to add CA certificate: %d", err);
+
+	}
+
+	err = tls_credential_add(CONFIG_NRF_CLOUD_SEC_TAG,
+				TLS_CREDENTIAL_PRIVATE_KEY,
+				private_key,
+				sizeof(private_key));
+	if (err < 0) {
+		LOG_ERR("Failed to add private key: %d", err);
+	}
+
+#endif
```

Comment:
Hmm, we already have `CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` which does nearly the same thing

It would be nice if we could just alter/extend that to work for CoAP

Or at least re-use the KConfig Symbol